### PR TITLE
Add memo and deleted to token info endpoint

### LINKED ISSUE #2522: Add memo to tokens REST API
### Problem

Currently GET token info REST API doesn't return `token memo`.
dragonglass  shows token empty memo value in `Profile Summary` section
see https://testnet.dragonglass.me/hedera/tokens/0.0.2276691

Screen Shot 2021-09-10 at 3 33 25 PM

### Solution

Add memo in the tokens endpoint `api/v1/tokens`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ijungmann**
Signed-off-by: Ian Jungmann <ian.jungmann@hedera.com>

**Description**:
Add token memo and deleted to `/tokens/{id}`

* Add memo and deleted to token info endpoint
* Update tests
* Update OpenApi

**Related issue(s)**:

Fixes #2522 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: V1.45.0__hip_24_pause_token.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-------------------
```
Comment (ijungmann) [inline V1.45.0__hip_24_pause_token.sql:1]: Remove, was stashed from other issue

**Comment by ijungmann** [comment]
Do a second pass on openApi to make sure everything was updated.

**Comment by ijungmann** [comment]
Question for tomorrow: should memo be part of NFT APIs?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/__tests__/specs/token-info-01-no-args.spec.json [hunk 2]
```diff
@@ -68,6 +72,7 @@
     "initial_supply": "1000000",
     "kyc_key": null,
     "max_supply": "9223372036854775807",
+    "memo": "token.0.0.1135",
```
[the remark this answers is not in the document]
Comment (ijungmann) [reply hedera-mirror-rest/__tests__/specs/token-info-01-no-args.spec.json:75]: Sure, `deleted` makes sense, I will add it.  I don't see anything else particularly useful for `token`

File: hedera-mirror-rest/__tests__/specs/tokens-04-token-range.spec.json [hunk 3]
```diff
@@ -60,12 +64,14 @@
...
         "symbol": "ORIGINALRDKSE",
         "admin_key": null,
+        "memo": "token.0.0.1135",
         "type": "FUNGIBLE_COMMON"
       },
       {
         "token_id": "0.7.25301",
         "symbol": "MIRRORTOKEN",
         "admin_key": null,
+        "memo": "token.0.7.25301",
```
[the remark this answers is not in the document]
Comment (ijungmann) [reply hedera-mirror-rest/__tests__/specs/tokens-04-token-range.spec.json]: Removed here, only added to token info now.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**